tla-bmcmt/src/main/scala/at/forsyte/apalache/tla/bmcmt/config/CheckerExceptionAdapter.scala

```diff
@@ -27,8 +27,8 @@ class CheckerExceptionAdapter @Inject()(sourceStore: SourceStore,
     case err: SanyException =>
       NormalErrorMessage("Error by TLA+ parser: " + err.getMessage)
 
-    case err: TLCConfigurationError =>
-      NormalErrorMessage(err.getMessage)
+    case err: ConfigurationError =>
+      NormalErrorMessage("Configuration error (see the manual): " + err.getMessage)
```

Comment:
Perhaps we can add the link to the relevant section of the manual here?

Reply:
Yeah, I was thinking about providing the link. The only problem is that the link is quite long.